server/public/model/feature_flags.go

```diff
@@ -70,7 +70,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.ConsumePostHook = false
 	f.CloudAnnualRenewals = false
 	f.CloudDedicatedExportUI = false
-	f.ChannelBookmarks = false
+	f.ChannelBookmarks = true
```

Comment:
That feels a bit odd to make it generally available now since I would've figured we'd want to let this part marinate for a while, but that's obviously not my call